src/System.IO.Pipes/src/System.IO.Pipes.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I didn't modify this line. Why is it showing up as modified? Is it the BOM character?

Reply:
```suggestion
<Project Sdk="Microsoft.NET.Sdk">
```